### DISCUSSION TITLE
SetGammaFallback()

### DISCUSSION THREAD

**Discussion by fernandafenelon** [JSBSim-Team/jsbsim#567] [Q&A] [answered]
Hello everyone,

how can I use the SetGammaFallback() for trimming in python?
http://jsbsim.sourceforge.net/JSBSim/classJSBSim_1_1FGTrim.html#a1bca232d03d3435dd38d9b1020b0b94b

**Comment by bcoconni** [comment, marked as answer]
Hi @fernandafenelon,

First of all, our developers docs has moved to https://jsbsim-team.github.io/jsbsim/ so you should use that address to get the last up to date docs (Docs on SourceForge are no longer maintained).

That being said, I am afraid that `FGTrim::SetGammaFallback()` is currently not available for the Python module. What would be your usage for this feature ?
